extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -493,6 +493,12 @@ private void uiStyle() {
 				extensionPanel.setOpaque(true);
 				extensionPanel.setBackground(new Color(46, 46, 46));
 			}
+
+			nameLabel.setToolTipText("The name of the application used in gradle");
+			packageLabel.setToolTipText("The Java package name of the application");
+			gameClassLabel.setToolTipText("The name of the main Java class implementing ApplicationListener");
+			destinationLabel.setToolTipText("The root directory of the project to write the generated files to");
```

Comment:
This is phrased very awkwardly. If I didn't have the context that it was the destination directory, I would be lost. How about "Where this will generate the project's root directory"?